doc/schemas/gov_openapi.json

```diff
@@ -66,6 +66,28 @@
         ],
         "type": "object"
       },
+      "EntityId": {
+        "pattern": "^[a-f0-9]{64}$",
+        "type": "string"
+      },
+      "EntityId_to_Script": {
+        "items": {
+          "items": {
+            "oneOf": [
+              {
+                "$ref": "#/components/schemas/EntityId"
+              },
+              {
+                "$ref": "#/components/schemas/Script"
+              }
+            ]
+          },
+          "maxItems": 2,
+          "minItems": 2,
+          "type": "array"
+        },
+        "type": "array"
+      },
```

Comment:
I think it should be possible to convert this `std::map` to an object (since the keys are strings), but since we wrap them in another type `nlohmann::json` will produce an array of pairs, which at least we correctly describe here.

Reply:
This is another "more of a comment than a suggestion", don't think there's any way around this with our current JSON abstractions.

Reply:
Good spot. I didn't pay attention to this as we already had something similar when `MemberId`s were `uint64_t` (see `uint64_to_Script` on left-hand side of diff). Do you think there's a quick fix for this?

Reply:
I don't think there's a quick fix, it should be possible with some explicit template instantiations but not worth investigating this now. Can you just check that this array-of-pairs is actually what's produced when we convert one of these maps to JSON? I notice that the nlohmann docs say any type "whose keys can construct an std::string" will convert to a JSON object, so its possible that your `operator std::string` on `EntityID` actually produces an object in the serialisation (in which case this auto-generated schema is wrong...).

Reply:
As discussed, because of the `operator std::string()`, this serialised the map as an object. I've removed the operator and used `.value()` instead so that the schema is in line with what the serialisation code actually does.